BlakeDonn/ideabox-boilerplate
# Add Predefined Assets to the Page

#### What’s this PR do? 
1. Creates a Assets directory
2. Adds Predefined assets to that directory
3. Adds assets to their respective places within the HTML structure
4. Sizes the assets in CSS so they don't get so big.

#### Where should the reviewer start?
Assets folder, `index.html`, the `img` tag styling on `styles.css`

#### How should this be manually tested?
Confirm that the proper assets are in the assets folder and that they are displayed in roughly the right places on the page when you open it in Terminal.
